packages/pluggableWidgets/area-chart-web/src/__tests__/AreaChart.spec.tsx

```diff
@@ -0,0 +1,135 @@
+import { createElement } from "react";
+import { ChartWidget } from "@mendix/shared-charts";
+import {
+    dynamicValue,
+    // dynamicValue,
```

Comment:
comment, duplication

Reply:
As there is no comments anymore i am solving this discussion